docs/changelogs/6.2.0_Changelog.rst

```diff
@@ -124,8 +123,7 @@ pupmod-simp-mcafee
 pupmod-puppetlabs-java_ks
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* All modules that depended on this functionality have been removed from the
-  distribution and this dangling dependency is also being removed.
+* This RPM has been removed, as it is no longer a dependency of any SIMP tools.
```

Comment:
'SIMP tools' is not correct.  We do not refer to Puppet modules as tools. This module has been removed because the modules that require it are no longer packaged by SIMP.